core/pkg/resultshandling/reporter/v2/container.go

```diff
@@ -0,0 +1,50 @@
+package v2
+
+import (
+	"fmt"
+	"github.com/kubescape/opa-utils/reporthandling"
+	"hash/fnv"
+)
+
+const (
+	ContainerApiVersion string = "container.kubscape.cloud"
+	ContainerKind       string = "Container"
+)
+
+type Container struct {
+	ImageTag   string            `json:"imageTag"`
+	ImageHash  string            `json:"imageHash"` //just in kind=pod imageHash:
+	ApiVersion string            `json:"apiVersion"`
+	Kind       string            `json:"kind"`
+	Metadata   ContainerMetadata `json:"metadata"`
+}
+
+type ContainerMetadata struct {
+	Name             string `json:"name"`
+	Namespace        string `json:"namespace"`
+	ParentKind       string `json:"parentKind"`
```

Comment:
Hi, I think we should have an object representing the parent (similar to the `ownerReferences` object)